extensions/lifecycle/objectProcessor/LifecycleObjectProcessor.js

```diff
@@ -94,9 +93,11 @@ class LifecycleObjectProcessor extends EventEmitter {
         };
     }
 
-    getConsumerParams() {
+    getConsumerParams(isTransitionProcessor = false) {
```

Comment:
instead of adding the topic here, it should be added/overriden directly in the derived class: e.g. in `transitionProcessor.getConsumerParams()`